pkg/volume/flexvolume/plugin.go

```diff
@@ -34,8 +34,7 @@ import (
 )
 
 const (
-	flexVolumePluginName       = "kubernetes.io/flexvolume"
-	flexVolumePluginNamePrefix = "flexvolume-"
+	flexVolumePluginName = "kubernetes.io/flexvolume"
```

Comment:
though this is an issue, I am not sure if we would influence the current plugin's actions when we directly change the plugin name